packages/webapp/webapp_client.js

```diff
@@ -1,18 +1,21 @@
+import find from "lodash.find";
+
 export const WebApp = {
   _isCssLoaded() {
     if (document.styleSheets.length === 0) {
       return true;
     }
 
-    return _.find(document.styleSheets, sheet => {
-      if (sheet.cssText && ! sheet.cssRules) { // IE8
-        return ! sheet.cssText.match(/meteor-css-not-found-error/);
+    return find(document.styleSheets, sheet => {
```

Comment:
Do we gain anything from `lodash`'s `find` versus converting `document.styleSheets` to an array with `Array.from` and using the native `Array.prototype.find`?

```js
Array.from(document.styleSheets).find(sheet => {
...
```

Reply:
Or even just `Array.prototype.find.call(document.styleSheets, sheet => {`

Reply:
happily